cmd/access/node_builder/access_node_builder.go

```diff
@@ -202,6 +202,7 @@ func DefaultAccessNodeConfig() *AccessNodeConfig {
 			EventFilterConfig:       state_stream.DefaultEventFilterConfig,
 			ResponseLimit:           state_stream.DefaultResponseLimit,
 			HeartbeatInterval:       state_stream.DefaultHeartbeatInterval,
+			MaxRegisterIDsPerMsg:    state_stream.DefaultMaxRegisterIdsPerMsg,
```

Comment:
I find the `PerMsg` a bit confusing, how about?
```suggestion
			RegisterIDsRequestLimit:    state_stream.DefaultMaxRegisterIdsPerMsg,
```